crates/evm/src/executor/inspector/cheatcodes/wallet.rs

```diff
@@ -75,14 +81,21 @@ fn create_wallet(private_key: U256, label: Option<String>, state: &mut Cheatcode
     let pub_key_x = pub_key.x().ok_or("No x coordinate was found")?;
     let pub_key_y = pub_key.y().ok_or("No y coordinate was found")?;
 
-    let pub_key_x = U256::from(pub_key_x.as_slice());
-    let pub_key_y = U256::from(pub_key_y.as_slice());
+    let pub_key_x = U256::from_be_bytes((*pub_key_x).into());
+    let pub_key_y = U256::from_be_bytes((*pub_key_y).into());
 
     if let Some(label) = label {
-        state.labels.insert(addr, label);
+        state.labels.insert(addr.to_alloy(), label);
     }
 
-    Ok((addr, pub_key_x, pub_key_y, private_key).encode().into())
+    Ok(DynSolValue::Tuple(vec![
```

Comment:
same here @DaniPopes looks like we can get away with some helper trait impls for composite types of dynsolvalues

Reply:
Ideally this would still be `.abi_encode()` but coming from `alloy_sol_types` static encoder, which is what we discussed previously. Otherwise, to use it you'd have to manually specify the type `<sol!(types...)>::encode((values...))`.